db/seeds/development_users.rb

```diff
@@ -3,12 +3,13 @@
   email: "roda@dxw.com"
 )
 partner_org_user = User.find_or_initialize_by(
-  name: "artner Org User",
+  name: "Partner Org User",
   email: "roda+dp@dxw.com"
 )
 
-administrator.password = "LlEeTtMmEeIiNn!1"
-partner_org_user.password = "LlEeTtMmEeIiNn!1"
+[administrator, partner_org_user].each do |user|
+  user.update(password: "LlEeTtMmEeIiNn!1", otp_required_for_login: false)
+end
```

Comment:
Tried using `update_all` here but it didn't work - I think because this is a regular array, rather than an Active Record Relation/Collection

Reply:
We've got a bit of a mix of approaches going on here - previously we set the fields individually and saved at the end (lines 16 & 19) whereas here we're doing a couple of saves. Feel free to ignore me here, but I wonder if it's worth picking one approach here: save at the end _or_ update all in one go.